SkietbaanBE/SkietbaanBE/Controllers/FeaturesController.cs

```diff
@@ -32,5 +32,23 @@ public User Search(string username)
             }
             return null;
         }
+
+        //// PUT: api/User/Update
+        [HttpPut]
+        [ActionName("Update")]
+        public async Task<IActionResult> PutUserMember([FromBody] User user)
+        {
+            User dbUser = null; //assume user does not exist
+            dbUser = _context.Users.Where(u => u.Username == user.Username)
```

Comment:
If you are looking for a user in the db and you have the id then use that in the where instead as it has indexing while username does not.  Unless you set up an index for username but you did not.